netd.yaml

```diff
@@ -76,11 +76,11 @@ data:
               "type": "host-local",
               "ranges": [
                 [
-                  {"subnet": podCidr}
-                ]
+                  {"subnet": @podCidr}
```

Comment:
similar to what you introduced, maybe just replace [{"subnet": @podCidr}] as @ipv4Subnet